### DISCUSSION TITLE
Installed on Debian 11 but cannot log in

### DISCUSSION THREAD

**Discussion by hunnabjorn** [inventree/InvenTree#4756] [Q&A]
I have installed Debian 11.6 in a virtual machine and then installed InvenTree using the default install.sh script.
After installation I have tried all kind of ways to log in on the web page.
I have re-installed several times incl. removing /etc/inventree library to start fresh.
Every time I try to login as admin in the web page I get a "The username and/or password you specified are not correct." back.

I have initialized a database (I think) and have now set the environment variables for admin user name as well as admin password and at the end of the installation process a text tell me this is the credentials to use.
I have also tried to use the dynamic password in the /etc/inventree library.

What could be wrong or what else could I try?

**Comment by matmair** [comment]
Hi there @hunnabjorn - this sounds like an installer issue. Could you open [an issue](https://github.com/inventree/InvenTree/issues/new?assignees=&labels=question%2Ctriage%3Anot-checked%2Csetup&template=install.yaml) and fill out the template there? That makes debugging a lot easier for us/me. thanks
